deployments/deploy.go

```diff
@@ -73,3 +68,29 @@ func doDeployment(deploymentConfig engine.Deployment) error {
 		return nil
 	}
 }
+
+func runDeploymentScript(context string, deploymentCommand []string, version string, p func(string, ...any)) error {
+	p("running %s-deployment command %s...", context, strings.Join(deploymentCommand, " "))
+	deploymentCommand = append([]string{"bash"}, deploymentCommand...)
+	cmd := exec.Command("/usr/bin/env", deploymentCommand[0:]...)
+	cmd.Env = os.Environ()
+	cmd.Env = append(cmd.Env, fmt.Sprintf("VERSION=%s", version))
+	output, err := cmd.StdoutPipe()
+	if err != nil {
+		return err
+	}
+
+	if err = cmd.Start(); err != nil {
+		return err
+	}
+
+	scanner := bufio.NewScanner(output)
+	for scanner.Scan() {
+		fmt.Println(scanner.Text())
```

Comment:
Are we not using `p` here intentionally?

Reply:
Not intentionally. We could use `p`! What do you think? It's a trade-off between being as raw as possible in just forwarding the results of the script and being clear what deployment is causing the output

Reply:
I'd go for raw right now, and change it later if there's a great argument to unraw